src/wallet/coinselection.cpp

```diff
@@ -299,7 +300,7 @@ bool KnapsackSolver(const CAmount& nTargetValue, std::vector<OutputGroup>& group
  OutputGroup
 
  ******************************************************************************/
-
+// adds coin to the output group
 void OutputGroup::Insert(const CInputCoin& output, int depth, bool from_me, size_t ancestors, size_t descendants, bool positive_only) {
```

Comment:
Method comments should be in the class where they are declared, not where they are implemented. In this case here:
https://github.com/bitcoin/bitcoin/blob/master/src/wallet/coinselection.h#L92

Also, please use the Doxygen format for these comments so that they will appear in HTML documentation. Same comment for `EligibleForSpending` below
```
/**
 * Adds coin to the output group.
 */
```